stable/artifactory-ha/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [10.0.15] - Aug 4, 2020
```

Comment:
Please add this to the README

Reply:
we already have it  - https://github.com/jfrog/charts/blob/master/stable/artifactory/README.md#artifactory-upgrade-from-6x-to-7x-app-version